aiida_common_workflows/workflows/relax/abinit/generator.py

```diff
@@ -23,8 +23,22 @@ class AbinitRelaxInputsGenerator(RelaxInputsGenerator):
     _default_protocol = 'moderate'
     _calc_types = {'relax': {'code_plugin': 'abinit', 'description': 'The code to perform the relaxation.'}}
     _relax_types = {
-        RelaxType.ATOMS: 'Relax only the atomic positions while keeping the cell fixed.',
-        RelaxType.ATOMS_CELL: 'Relax both atomic positions and the cell.'
+        RelaxType.NONE:
+        'Fix the atomic positions and volume and shape of the cell.',
```

Comment:
Fix the atomic positions and shape of the cell.

Reply:
This fixes everything right? Could say that we fix the atomic positions and cell (cell = volume + shape), or we're fixing the atomic positions, shape, and volume.